create_votes.rb

```diff
@@ -0,0 +1,28 @@
+require "./db/setup"
+require "./lib/all"
+require "./lib/time"
+
+require './create_songs'
+require './create_users'
+
+def random_vote
+  x = {
+    0 => -1,
+    1 => 1
+  }
+  x[rand(2)]
+end
+
+User.all.each do |user|
+  Song.all.each do |song|
+    begin
+      Vote.create!(
+        song_id: song.id,
+        user_id: user.id,
+        value: random_vote,
+        placed_at: week
+      )
+    rescue ActiveRecord::RecordInvalid
+    end
+  end
+end
```

Comment:
All these little developer utility scripts are great, and please do keep writing them - but they are a little daunting to understand which one is useful for what. A new developer coming on the project would really appreciate these being 1) in a folder somewhere and 2) documented a little more (in the README, or in comments)